### ISSUE TITLE
Tabs - <nav> examples should show tabs as links

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1585]
In the examples we have where tabs are wrapped with a `<nav>` element, these tabs should be implemented as links.

This update will probably require some updates to the css to support either a link or button element. 

These examples should also include notes in the documentation section explaining that when links are used as tabs, the `<a>` elements should be include `href` attributes with valid urls that enable users to navigate to that tab using the url associated with the tab.
